sdk/communication/communication-sms/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.2.0-beta.1 (2024-04-10)
+
+### Features Added
+
+- Added DeliveryReportTimeoutInSeconds to smsSendOptions.
+
 ## 1.1.1 (Unreleased)
```

Comment:
Can we remove this entire section for 1.1.1 as it doesn't contain anything?

Reply:
removed